android/app/src/main/java/com/pillarproject/wallet/MainApplication.java

```diff
@@ -60,6 +61,11 @@ public void onCreate() {
         // react-native-async-storage custom max storage
         long storageSizeMax = 60L * 1024L * 1024L; // 60 MB
         com.reactnativecommunity.asyncstorage.ReactDatabaseSupplier.getInstance(getApplicationContext()).setMaximumSize(storageSizeMax);
+
+        new RNInstabugReactnativePackage
+            .Builder(getString(R.string.instabug_token), MainApplication.this)
+            .setInvocationEvent("shake")
```

Comment:
I wonder if we should add some option in the menu to manually trigger the instaback dialog. Shake gesture is kinda hidden and not evident.

Reply:
@mdjastrzebski just to let you know that for now, we just want to test out with the shake gesture. This is to prevent any clashes with the existing Intercom support system, however once we have successfully tested Instabug with a select few users via the shake mechanism, then we'll replace Intercom and make the invocation of this feature more prominent on the home screen.

Reply:
In addition, with the default configuration, Instabug shows a message explaining how it can be accessed when launching the app for the first time: https://docs.instabug.com/docs/react-native-onboarding-experience so it's not completely hidden.